azurerm/resource_arm_storage_account.go

```diff
@@ -1146,5 +1225,49 @@ func flattenAndSetAzureRmStorageAccountSecondaryEndpoints(d *schema.ResourceData
 	d.Set("secondary_table_endpoint", tableEndpoint)
 	d.Set("secondary_table_host", tableHost)
 
+	var webEndpoint, webHost string
+	if secondary != nil {
+		if v := secondary.Web; v != nil {
```

Comment:
I wonder if it would be worthwhile to add a helper function to this file 🤔 
```go
if secondary != nil {
  webEndpoint, webHost err = parseEndPoint(secondary.Web)
  if err != nil { return err }
}
```

Reply:
Yeah, I can refactor to add one in.